open_academy/views/res_partner_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="partner_action" model="ir.actions.act_window">
+        <field name="name">partner.view.form</field>
+        <field name="res_model">res.partner</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <menuitem id="open_academy_root_partner_menu" name="Partners" sequence="1"/>
+
+    <menuitem id="partner_menu" name="Partners" sequence="1"
+    action="partner_action" parent="open_academy_root_partner_menu"/>
+
+    <record id="res_partner_view_form" model="ir.ui.view">
```

Comment:
When a record is inherited, the id used must be the same as the inherited record.

```suggestion
    <record id="view_partner_form" model="ir.ui.view">
```